python/ray/data/_internal/stats.py

```diff
@@ -774,6 +774,13 @@ def to_string(self) -> str:
             out += (
                 "* Batch iteration time breakdown (summed across prefetch threads):\n"
             )
+            if self.wait_time.get():
+                out += "    * In block fetching: {} min, {} max, {} avg, {} total\n".format(
+                    fmt(self.wait_time.min()),
+                    fmt(self.wait_time.max()),
+                    fmt(self.wait_time.avg()),
+                    fmt(self.wait_time.get()),
+                )
             if self.get_time.get():
```

Comment:
`In ray.get(): ...` is very unintuitive to our user. We should have a better summary print here.

Reply:
Agreed